CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix redis db.statements to be sanitized by default
+  ([#1778](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1776))
```

Comment:
You had a typo, it should be `#1776` instead of `#1778`

Reply:
Thanks!